src/components/ProjectTable/ProjectTableRow.js

```diff
@@ -15,7 +15,6 @@ const ProjectTableRow = ({
   return (
     <div
       className={`project-table-row ${isOpen ? "is-open" : "is-closed"}`}
-      id={project.slug}
```

Comment:
Fix `An update to Link inside a test was not wrapped in act` warning in the console